web/docs/language/features.md

```diff
@@ -443,6 +443,183 @@ import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
   }
 ```
 
+## Jobs
+
+If you have tasks that you do not want to handle as part of the normal request-response cycle, Wasp allows you to make that function a `job` and it will gain some "superpowers." Jobs will persist between server restarts, can be retried if they fail, and they can even be delayed until the future (or have a recurring schedule)! Some examples where you may want to use a `job` include sending an email, making an HTTP request to some external API, or doing some nightly calculations.
```

Comment:
Nice!!!

Should we epmhasize these are executed on the server, that it is server logic?

Reply:
Yep, I can add that in.